### DISCUSSION TITLE
How to delete a workspace?

### DISCUSSION THREAD

**Discussion by armedjuror** [fief-dev/fief#41] [Q&A]
I created a workspace. How to delete the same.

**Comment by frankie567** [comment]
Hi @armedjuror, welcome to Fief 👋 

Unfortunately, that's not something available through the API or admin UI right now.

Is it on the Cloud version? If so, I can delete it for you. You just need to give me your workspace name.

**Comment by michaldev** [comment]
I also need this feature.
